daemon/main.c

```diff
@@ -532,6 +532,28 @@ main_terminate(
 //  Set process scheduling policy/priority and Lock resident memory.
 //
 
+static int move_to_root_cgroup(void) {
```

Comment:
The return type of this function could be `void`. It currently returns an integer that is then immediately ignored. Changing the return type to void would simplify the code a bit.